React: update 'render' and 'component' descriptions (#5676)
DevExpress/devextreme-documentation: concepts/50 React Components/40 Component Configuration Syntax/55 Markup Customization/8 Using the Template Component.md

> Several properties are not implemented as nested configuration components (**columns[].**[editorOptions](/api-reference/_hidden/GridBaseColumn/editorOptions.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#editorOptions') in the DataGrid, item's [editorOptions](/api-reference/10%20UI%20Components/dxForm/5%20Item%20Types/SimpleItem/editorOptions.md '/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#editorOptions') in the Form, **items[]**.[options](/api-reference/_hidden/dxToolbarItem/options.md '/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#options') in the Toolbar). These properties do not have the `render` or `component` property to which you would pass your rendering function or custom component. However, you can still customize the markup &mdash; using the `Template` element.

Reviewer: ```suggestion
Several properties are not implemented as nested configuration components (**columns[].**[editorOptions](/api-reference/_hidden/GridBaseColumn/editorOptions.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#editorOptions') in the DataGrid, item's [editorOptions](/api-reference/10%20UI%20Components/dxForm/5%20Item%20Types/SimpleItem/editorOptions.md '/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#editorOptions') in the Form, **items[]**.[options](/api-reference/_hidden/dxToolbarItem/options.md '/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#options') in the Toolbar). These properties do not have a `Render` or `Component` property where you would pass your rendering function or custom component. However, you can still customize the markup. Use the `Template` element.
```

> The `Template` element declares a named template. Its `name` property should be assigned to a `template` property of the UI component that uses the `Template`. The template's markup can be specified as follows:

Reviewer: ```suggestion
The `Template` element declares a named template. Its `name` property value should be assigned to a `Template` property of the UI component. You can specify the template's markup as follows:
```

> Several properties are not implemented as nested configuration components (**columns[].**[editorOptions](/api-reference/_hidden/GridBaseColumn/editorOptions.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#editorOptions') in the DataGrid, item's [editorOptions](/api-reference/10%20UI%20Components/dxForm/5%20Item%20Types/SimpleItem/editorOptions.md '/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#editorOptions') in the Form, **items[]**.[options](/api-reference/_hidden/dxToolbarItem/options.md '/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#options') in the Toolbar). These properties do not have a `Render` or `Component` property where you would pass your rendering function or custom component. However, you can still customize the markup. Use the `Template` element.

Reviewer 2: ```suggestion
Several properties are not implemented as nested configuration components (**columns[].**[editorOptions](/api-reference/_hidden/GridBaseColumn/editorOptions.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#editorOptions') in the DataGrid, item's [editorOptions](/api-reference/10%20UI%20Components/dxForm/5%20Item%20Types/SimpleItem/editorOptions.md '/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#editorOptions') in the Form, **items[]**.[options](/api-reference/_hidden/dxToolbarItem/options.md '/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#options') in the Toolbar). These properties do not have a `Render` or `Component` property where you would pass your rendering function or custom component. However, you can still use the `Template` element to customize the markup.
```
